server/models/mongodb/MongoRepository.js

```diff
@@ -23,23 +23,26 @@ const getMembersFromDb = async (memberId = null) => {
 }
 
 /**
- * Saves a member to the database.
+ * Saves a `memberObject` to the database.
+ *
```

Comment:
Is having a space between these comments necessary?

Reply:
Good point! Let's combine those together.